mappings/net/minecraft/client/gui/screen/multiplayer/MultiplayerWarningScreen.mapping

```diff
@@ -1,9 +1,9 @@
 CLASS net/minecraft/class_4749 net/minecraft/client/gui/screen/multiplayer/MultiplayerWarningScreen
 	FIELD field_21842 parent Lnet/minecraft/class_437;
-	FIELD field_21843 header Lnet/minecraft/class_2561;
-	FIELD field_21844 message Lnet/minecraft/class_2561;
-	FIELD field_21845 checkMessage Lnet/minecraft/class_2561;
-	FIELD field_21846 proceedText Lnet/minecraft/class_2561;
+	FIELD field_21843 HEADER Lnet/minecraft/class_2561;
+	FIELD field_21844 MESSGE Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	FIELD field_21844 MESSAGE Lnet/minecraft/class_2561;
```

Reply:
Fixed in 9032c68e7b6124111bc7f1f7f0db3390651f07a2